change/@fluentui-react-drawer-ecbe0ab7-6a66-465a-ac31-daf11345c21c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30866&commitId=201cec719e35c0a01c779773c915740361081a18&env=prod&clientType=FLUENTUI)


  <details><summary><b>Checkbox</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checkbox Example Checked.chromium.png|25| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>